tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -266,6 +266,10 @@ public void ignoredOperation() {
 
   }
 
+  public void throwError() {
+    throw new AssertionError();
```

Comment:
This isn't fatal error as far as reactor is concerned, should we test with a fatal JVM error too? e.g. StackOverflow?